github/maintenance_branch/create_maintenance_branch_sle15.rb

```diff
@@ -0,0 +1,418 @@
+#!/usr/bin/env ruby
+
+# This script creates a new maintenance branch for all YaST packages.
+#
```

Comment:
Please describe the difference to the `create_maintenance_branch.rb` script we know in which cases to use it.